GNUmakefile

```diff
@@ -1,22 +1,12 @@
-TESTS = $(wildcard test/*.lean)
-
-# Ensure that panics actually cause the tests to fail
-export LEAN_ABORT_ON_PANIC=1
-
 .PHONY: all build test lint
 
 all: build test
 
 build:
 	lake build
 
-# Find all .lean files in test and subdirectories, replace .lean with .run
-TESTS := $(shell find test -type f -name '*.lean' | sed 's/\.lean$$/.run/')
-
-test: $(TESTS)
-
-%.run: build
-	lake env lean $(@:.run=.lean)
+test:
+	lake test
 
 lint: build
 	./.lake/build/bin/runLinter
```

Comment:
Separate from this PR, should this be using `lake exe runLinter`?

Reply:
#791